go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/grafana/beyla
 
-go 1.23
+go 1.23.2
```

Comment:
Not sure if relevant here, but perhaps we should double check if this has any implications with Alloy

Reply:
It was automatically added by `go mod tidy`. Let me just revert this line.